### DISCUSSION TITLE
2011/12/15/%E5%BC%82%E6%AD%A5%E8%B0%83%E7%94%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#350] [Announcements]
# 2011/12/15/%E5%BC%82%E6%AD%A5%E8%B0%83%E7%94%A8

https://labview.qizhen.xyz/

https://blog.qizhen.xyz/2011/12/15/%E5%BC%82%E6%AD%A5%E8%B0%83%E7%94%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

atwater
2012年02月9日 04:37 编辑
有这个程序的代码吗?

回复
ruanqizhen
2012年02月14日 09:14 编辑
演示程序代码在这里下载：https://decibel.ni.com/content/docs/DOC-20916

回复
